<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window6</class>
 <widget class="QWidget" name="window6">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="connectBT">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>124</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>372</height>
          </size>
         </property>
         <property name="text">
          <string>未连接</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="subBT">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>124</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>372</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">image:url(:/subscribe.png);
image-position: left;
text-align:right;
padding-left:30px;
padding-right:30px;</string>
         </property>
         <property name="text">
          <string>订阅</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_1">
         <property name="title">
          <string>状态显示框</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QTextBrowser" name="stateTB"/>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="clearPB1">
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>心跳接收框</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QTextBrowser" name="heartbeatTB"/>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="clearPB2">
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>消息接收框</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="0">
           <widget class="QPushButton" name="clearPB3">
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QTextBrowser" name="receiveTB"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>消息发送框</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QTextBrowser" name="sendTB"/>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="clearPB4">
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>124</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1800</width>
         <height>124</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">image:url(:/back.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
